openhtf/util/validators.py

```diff
@@ -87,6 +87,10 @@ def __init__(self, minimum=None, maximum=None):
       self.maximum = maximum
 
     def __call__(self, value):
+      # Check for nan
+      import math
```

Comment:
copy the usage of `numbers`, `import math` at the top, pass it to `Validators.__init__()`, stash it on self in that constructor, and reference it here as `self.math_module.isnan()`.  Sorry this is tricky, it's done to provide `__getattr__` behavior at the module level, and this is actually the officially suggested way to do this.